frontend/src/components/ProjectList/components/Dataset/components/MoreInformation/index.tsx

```diff
@@ -0,0 +1,29 @@
+import React, { FC } from "react";
+import { Project } from "src/common/entities";
+import { StyledAnchor, Wrapper } from "./style";
+
+interface Props {
+  links: Project["links"];
+}
+
+const MoreInformation: FC<Props> = ({ links }) => {
+  return (
+    <Wrapper>
+      {links.map(link => {
+        return (
+          <StyledAnchor
+            key={link.url}
+            href={link.url}
+            target="_blank"
+            rel="noopener"
```

Comment:
no need for adding `noreferrer`, since it's a security fallback for IE11 <, which I don't think we should support. 

The upside for not adding `noreferrer` is that out linked pages will know our website referred the users to their pages!